shared/engine/transport-shared.js

```diff
@@ -39,6 +140,22 @@ class TransportShared extends RobustTransport {
       return new Error(JSON.stringify(err))
     }
   }
+
+  invoke (arg: Object, cb: any) {
+    // args needs to be wrapped as an array for some reason so lets just do that here
+    const wrappedArgs = {
+      ...arg,
+      args: [arg.args || {}],
+    }
+
+    const wrappedInvoke = _wrap((args) => {
```

Comment:
🐑 Why not use the closed over `wrappedArgs`. Then this doesn't take an argument and you'll have one fewer thing in this function with the word `args`